pkg/controller/kieapp/kieapp_controller_test.go

```diff
@@ -198,60 +198,6 @@ func TestCreateRhdmImageStreams(t *testing.T) {
 	assert.Equal(t, fmt.Sprintf("registry.redhat.io/rhdm-7/rhdm%s-decisioncentral-openshift:1.0", cr.Spec.CommonConfig.Version), isTag.Tag.From.Name)
 }
 
-func TestCreateRhdmTechPreviewImageStreams(t *testing.T) {
```

Comment:
@spolti rather than delete these tests, should we instead modify them for datagrid images?

Reply:
Agreed for the second test, as there is no thec preview image anymore, should we keep this test?

Reply:
i think both are valuable... one checks for latest tag, and the other checks for `1.1` versioned tag?

Reply:
ok, make sense.